firestore/FirestoreSwiftUIExample/Views/StarsView.swift

```diff
@@ -43,8 +43,33 @@ struct StarsView: View {
   }
 }
 
+struct MutableStarsView: View {
```

Comment:
nit: Maybe name this `StarsInputView` instead, to make it more clear it's used solely as an input field presented to the user.